### ISSUE TITLE
exported members have generated names

### ISSUE THREAD

**Issue by FelschR** [dschnelldavis/angular2-json-schema-form#99]
After updating to the most recent version I get the following errors when importing one of e.g.:
addClasses, inArray, JsonPointer, toTitleCase

```sh
Module '"C:/Users/felix_schroeter/Documents/Projects/carepal-app/node_modules/angular2-json-schema-form/angular2-json-schema-form"' has no exported member 'addClasses'.
```

Importing JsonSchemaFormService works.

When using the import in TypeScript I can see the exported members ea, eb, eba, ... which seem to be generated by Angular.
So something is definitely wrong here.

**Comment by FelschR** [comment]
I checked a few things. The index.d.ts contains the following:
```ts
export { JsonSchemaFormComponent } from './src/json-schema-form.component';
export { JsonSchemaFormService } from './src/json-schema-form.service';
export { JsonSchemaFormModule } from './src/json-schema-form.module';
```

before the update its content was:
```ts
export * from './src/shared/convert-json-schema.functions';
export * from './src/shared/validator.functions';
export * from './src/shared/utility.functions';
export * from './src/shared/jsonpointer.functions';
export * from './src/shared/json.validators';
export * from './src/shared/json-schema.functions';
export * from './src/shared/form-group.functions';
export * from './src/shared/layout.functions';
export { JsonSchemaFormComponent } from './src/json-schema-form.component';
export { JsonSchemaFormService } from './src/json-schema-form.service';
export { JsonSchemaFormModule } from './src/json-schema-form.module';
```

Changing it back didn't help.

Did the way of importing the shared exports change?

**Comment by FelschR** [comment]
Ok, I found the solution:
```ts
import { buildTitleMap } from "angular2-json-schema-form/src/shared";
```

**Comment by dschnelldavis** [comment]
You're right - the functions weren't being exported correctly. I've fixed it in the latest version (0.5.0-alpha.10), so it should be working again.

So this should now work:
```javascript
import { buildTitleMap } from "angular2-json-schema-form";
```

If it doesn't for some reason, you can also import the function directly from its source like so (though this should not be necessary):
```javascript
import { buildTitleMap } from "angular2-json-schema-form/src/shared/layout.functions";
```

Let me know if you have any additional problems with this.

**Comment by FelschR** [comment]
0.5.0-alpha.10 is working fine, thanks!
